.github/ISSUE_TEMPLATE/A-submit-software-for-review.md

```diff
@@ -4,6 +4,10 @@ about: Use this template to submit software for review
 
 ---
 
+<!---
+In the section that immediately follows, please enter the 2 values of (1) submitting author GitHub handle; and (2) Repository URL. Values for additional package authors may also be specified. Values for each must be placed between the two HTML sections, as illustrated for the "author1" value. The HTML symbols defining each variable must be left as they are - so the submitting author is specified by replacing "@github_handle" only, while leaving the enclosing HTML symbols as they are. This comment may be deleted once it has been read and understood.
```

Comment:
```suggestion
Below, please enter the values of (1) submitting author GitHub handle (replacing "@github_handle"); and (2) Repository URL (replacing "http:://repourl"). Values for additional package authors may also be specified replacing "@github_handle1," "@github_handle2" - delete these if not needed. DO NOT DELETE HTML symbols (e.g." <!--author1-->", "<!--repourl-->"). Replace "@github_handle" only.  This comment may be deleted once it has been read and understood.
```

An attempt at more clarity and brevity.   Please add `http://repourl` in the template - I don't think you have allowed me to edit the PR directly.